apps/src/gamelab/GameLab.js

```diff
@@ -379,10 +374,10 @@ GameLab.prototype.init = function(config) {
     showDebugWatch:
       config.level.showDebugWatch || experiments.isEnabled('showWatchers'),
     showDebugSlider: experiments.isEnabled('showDebugSlider'),
-    showAnimationMode: !config.level.hideAnimationMode || showCostumeTab,
+    showAnimationMode: !config.level.hideAnimationMode,
     startInAnimationTab: config.level.startInAnimationTab,
     allAnimationsSingleFrame:
-      config.level.allAnimationsSingleFrame || showCostumeTab,
+      config.level.allAnimationsSingleFrame || this.studioApp_.isUsingBlockly(),
```

Comment:
If we are setting this to true in the level config, do we need to have `this.studioApp_.isUsingBlockly()`?

Reply:
There is a potential that level builders could accidentally set this to true in a sprite lab level, but we want to force it to always be false in sprite lab.